pokemons/helpers.py

```diff
@@ -29,3 +41,32 @@ def pokemon_exists(name):
     if requests.get(url).status_code == 404:
         return False
     return True
+
+
+def get_pokemons_from_trainerteam(battle, trainer):
+    trainerteam = TrainerTeam.objects.get(
+        battle_related=battle,
+        trainer=trainer
+    )
+    return {
+        Pokemon.objects.get(name=trainerteam.pokemon_1),
+        Pokemon.objects.get(name=trainerteam.pokemon_2),
+        Pokemon.objects.get(name=trainerteam.pokemon_3),
+    }
+
+
+# task
+def save_pokeapi_pokemons():
+    url = requests.get(f'{POKEAPI_ROOT}/?limit=802')
+    poke_api = url.json()
+    for pokemon in poke_api['results']:
+        if not Pokemon.objects.filter(name=pokemon['name']).exists():
```

Comment:
There's a pretty neat programming pattern called [guard clauses](https://refactoring.guru/replace-nested-conditional-with-guard-clauses). Take a quick read at it and try to apply it to this block of code to make it less indented and easier to read :)

Reply:
Done!